### ISSUE TITLE
provide a way to link directly to one apprenticeship

### ISSUE THREAD

**Issue by jmmastey** [jmmastey/chicagoapprenticeships#2]
Since the http://chicagoapprenticeships site tends to have more concrete information about any given apprenticeship than the company's own hiring page, I'd like to be able to link people directly to a given apprenticeship.

A link might look something like http://chicagoapprenticeships.com/#launchpad, and when a user visited that link, that apprenticeship should be shown in a more focused way (not just alongside the other entries).

Questions / Concerns:
1. How can users grab the link? This may need to come after #1 so we can provide a new button.
2. Speaking of which, can we add some kind of social sharing button?
3. What should the entry look like while "inflated"? A modal is probably the smartest answer to this question.
4. Does it work cleanly on mobile?

**Comment by davidwkaiser** [comment]
Given the quantity of broken links or links with minimal content (a "careers" page for example), I imagine that the companies must change their addresses or content a lot. It would probably make sense to host info about the apprenticeships on this site, with contact info or a link to the companies' site. 

I thought launchpad looked nice, that could be a good template. Perhaps an eventual feature is a form the companies can use to update their own data. Or maybe the security concerns aren't worth it. Anyway, that's what I got for now. 

I think a social sharing button should be pretty doable.

**Comment by jmmastey** [comment]
Agreed, though I don't want to be responsible for keeping all information ever, given we're not a job site. That said, having those pages would also increase the total content on the site (at least according to Google), and I may have longer descriptions for most of the apprentice programs' curricula, and also interviews for several of those companies.
